### ISSUE TITLE
[Feature]: TUS Resumable Uploads

### ISSUE THREAD

**Issue by paulocastellano** [ToshY/BunnyNet-PHP#93]
### Description

It's possible to add TUS Resumable Uploads for large videos?

**Comment by ToshY** [comment]
Hey @paulocastellano 👋 

Thank you for your question. As I have no prior knowledge with TUS, I've started reading both the [bunny.net documentation](https://docs.bunny.net/reference/tus-resumable-uploads) as well as [tus.io faq](https://tus.io/faq#how-does-tus-work). 

It seems there is already an existing PHP implementation specifically for TUS: [ankitpokhrel/tus-php](https://github.com/ankitpokhrel/tus-php). Looking through these docs and codebase it seems to me that it goes way beyond an API call (and requires additional functionality like caching). 

Keep in mind that BunnyNet-PHP is "basically"[^1] just an API client for Bunny.net, and only has a handful of essential dependencies. The idea is to keep it that way, without adding more complexity than it actually needs to function properly.

That being said, I'm all ears if you have more knowledge on the subject or have other ideas that would make this easy to implement without the need of (many) additional dependencies.

[^1]: I wrote "basically" because I also included the `TokenAuthentication` class which is strictly speaking not part of the API; still a good (dependency-free) addition nonetheless.
